aws_lambda_powertools/utilities/parser/models/sns.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel
+from pydantic.networks import HttpUrl
+from typing_extensions import Literal
+
+
+class SqsMsgAttributeModel(BaseModel):
```

Comment:
Meant SNS instead of SQS?

```suggestion
class SnsMsgAttributeModel(BaseModel):
```

Reply:
fixed!